internal/move2kubeapi/move2kubeapi.go

```diff
@@ -254,7 +263,15 @@ func deleteTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 func startPlan(w http.ResponseWriter, r *http.Request) {
 	name := mux.Vars(r)["name"]
 
-	err := m2kapp.GeneratePlan(name)
+	keys, ok := r.URL.Query()["debug"]
```

Comment:
Do we want to convert it to a boolean here itself?

Reply:
Yes sure, I have converted it to a boolean.